lib/msal-common/src/url/UrlString.ts

```diff
@@ -39,7 +39,7 @@ export class UrlString {
             url = url.toLowerCase();
         }
 
-        if (url && !url.endsWith("/")) {
+        if (url && (url.charAt(url.length - 1) !== "/")) {
```

Comment:
Can we add an `endsWith` string util similar to startsWith?

Reply:
Done